drivers/frequency/hmc7044/hmc7044.c

```diff
@@ -804,6 +815,8 @@ int32_t hmc7044_init(struct hmc7044_dev **device,
 	dev->clkin1_vcoin_en = init_param->clkin1_vcoin_en;
 	dev->high_performance_mode_clock_dist_en =
 		init_param->high_performance_mode_clock_dist_en;
+	dev->high_performance_mode_pll_vco_en =
```

Comment:
This should be in a separate commit, with description, why it is needed.

Reply:
You are not using the latest version of no-OS - this has been removed by https://github.com/analogdevicesinc/no-OS/commit/1915ff0af51ee6b65470f6a4fa4a71bab9afa884

Reply:
Apologies, this snuck in due to a convoluted merge process on our end. Reverted.